src/rpc/claimrpchelp.h

```diff
@@ -0,0 +1,341 @@
+#define T_HEIGHT                        "height"
+#define T_VALUE                         "value"
+#define T_NAME                          "name"
+#define T_VALIDATHEIGHT                 "validAtHeight"
+#define T_NAMES                         "names"
+#define T_EFFECTIVEAMOUNT               "effectiveAmount"
+#define T_LASTTAKEOVERHEIGHT            "lastTakeoverHeight"
+#define T_SUPPORTS                      "supports"
+#define T_SUPPORTSWITHOUTCLAIM          "supportsWithoutClaim"
+#define T_TOTALNAMES                    "totalNames"
+#define T_TOTALCLAIMS                   "totalClaims"
+#define T_TOTALVALUE                    "totalValue"
+#define T_CONTROLLINGONLY               "controllingOnly"
+#define T_CLAIMTYPE                     "claimType"
+#define T_DEPTH                         "depth"
+#define T_INCLAIMTRIE                   "inClaimTrie"
+#define T_ISCONTROLLING                 "isControlling"
+#define T_INSUPPORTMAP                  "inSupportMap"
+#define T_INQUEUE                       "inQueue"
+#define T_BLOCKSTOVALID                 "blocksToValid"
+#define T_NODES                         "nodes"
+#define T_CHILDREN                      "children"
+#define T_CHARACTER                     "character"
+#define T_NODEHASH                      "nodeHash"
+#define T_VALUEHASH                     "valueHash"
+#define T_PAIRS                         "pairs"
+#define T_ODD                           "odd"
+#define T_HASH                          "hash"
+#define T_BID                           "bid"
+#define T_SEQUENCE                      "sequence"
+#define T_CLAIMSADDEDORUPDATED          "claimsAddedOrUpdated"
+#define T_SUPPORTSADDEDORUPDATED        "supportsAddedOrUpdated"
+#define T_CLAIMSREMOVED                 "claimsRemoved"
+#define T_SUPPORTSREMOVED               "supportsRemoved"
+#define T_ADDRESS                       "address"
+
+enum {
+    GETCLAIMSINTRIE = 0,
+    GETNAMESINTRIE,
+    GETVALUEFORNAME,
+    GETCLAIMSFORNAME,
+    GETCLAIMBYID,
+    GETTOTALCLAIMEDNAMES,
+    GETTOTALCLAIMS,
+    GETTOTALVALUEOFCLAIMS,
+    GETCLAIMSFORTX,
+    GETNAMEPROOF,
+    CHECKNORMALIZATION,
+    GETCLAIMBYBID,
+    GETCLAIMBYSEQ,
+    GETCLAIMPROOFBYID,
+    GETCLAIMPROOFBYSEQ,
+    GETCHANGESINBLOCK,
+};
+
+#define S3_(pre, name, def) pre "\"" name "\"" def "\n"
+#define S3(pre, name, def) S3_(pre, name, def)
+#define S1(str)  str "\n"
+
+#define NAME_TEXT      "                    (string) the name to look up"
+#define BLOCKHASH_TEXT "               (string, optional) get claims in the trie\n" \
+"                                                  at the block specified\n" \
+"                                                  by this block hash.\n" \
+"                                                  If none is given,\n" \
+"                                                  the latest active\n" \
+"                                                  block will be used."
+
+#define CLAIM_OUTPUT    \
```

Comment:
These are less readable than before. I actually used these quite often.

Reply:
I think is more readable now :) You see it on one place and there is no need to verify it's same or not and it's correct on every place or not.

Reply:
I was also looking at it through the PR lens. Will look at it in the code base too.

Reply:
Overall code base is way better to me, logic and help message are separate which makes logic to be easy follow, OTOH help message are minimize.